### PR TITLE
feat: add 1.12 release blog

### PR DESCRIPTION
**Pull request by realshuting**

### CODE REVIEW

File: content/en/blog/releases/1-12-0/index.md [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+* Updated frontend tech stack: Bun, Vue 3.x, Nuxt 3.x, Vuetify 3.x, Vite 5.x
+* Color Blind Themes
+* General performance improvements
+
+### Kyverno Playground
+
+The Kyverno playground with the support of ValidatingAdmissionPolicyBinding is now available for the new 1.12 release. You can check it out today via https://playground.kyverno.io/next/#/.
+
+## Closing
+We have achieved another significant milestone, thanks to the support and contributions from our incredible community. Alongside the highlighted new features, this release includes critical fixes and other important changes. For a comprehensive overview, please refer to the full release notes on GitHub.
```
Comment (realshuting) [inline content/en/blog/releases/1-12-0/index.md:310]:
```suggestion
We have achieved another significant milestone, thanks to the support and contributions from our incredible community. Alongside the highlighted new features, this release includes critical fixes and other important changes. For a comprehensive overview, please refer to the full [release notes](https://github.com/kyverno/kyverno/releases/tag/v1.12.0) on GitHub.
```

File: content/en/blog/releases/1-12-0/index.md [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+```
+
+_Note_: Depending on whether the data is sourced from Kubernetes resources or API calls, the JMESPath expression used to manipulate the data may vary. For Kubernetes resources, utilize `length(@)` to obtain the length of the array, while for API calls returning a list, use `items | length(@)`.
+
+In conclusion, Global Context in Kyverno empowers users to streamline policy management by centralizing and reusing data across policies, thereby enhancing efficiency and maintainability in Kubernetes environments.
+
+### Kyverno JSON In CLI
+
+Kyverno JSON was introduced around the same time as Kyverno 1.11.0, expanding Kyverno's capabilities beyond Kubernetes. With the latest 1.12 release, Kyverno JSON is now supported by the Kyverno CLI for validating any JSON payload.
+
```
[the remark this answers is not in the document]
Comment (eddycharly) [reply content/en/blog/releases/1-12-0/index.md:101]: 👀

**Review by vishal-chdhry — APPROVED** [review]

**Comment by chipzoller** [comment]
Link checker failed.

**Comment by vishal-chdhry** [comment]
@chipzoller The links are failing in https://github.com/kyverno/website/blob/main/content/en/resources/_index.md, one is a 500, so it might be that the server is down, other is `403`
Looking into it

**Pull request merged**